.eslintrc.js

```diff
@@ -71,5 +82,25 @@ module.exports = {
         returns: 'return'
       }
     }
-  }
+  },
+  overrides: [
+    {
+      files: ['**/*.md'],
+      processor: 'markdown/markdown'
+    },
+    {
+      files: ['**/*.vue live'],
```

Comment:
The `eslint-plugin-markdown` plugin will look at any markdown fence and associate the syntax with a filetype. So when it sees `vue live` it tells ESLint to lint that code block using the settings for `vue live` which didn't exist. So I defined it here and told it treat it as a vue file (`vue/.vue`). Then added some custom override rules for markdown fenced as vue live.